setup.py

```diff
@@ -23,7 +23,7 @@
 ]
 
 setup(name='LstGen',
-    version='0.5.4',
+    version='0.5.6',
```

Comment:
Please let me change this in an extra commit when I bump the version and update the release note